crates/swc_ecma_parser/src/lexer/mod.rs

```diff
@@ -1223,16 +1247,14 @@ impl<'a> Lexer<'a> {
                 if let Ok(ref mut cooked) = cooked {
                     cooked.push(c);
                 }
-
                 raw.push(c);
+                raw_slice_start = self.cur_pos();
             } else {
                 self.bump();
 
                 if let Ok(ref mut cooked) = cooked {
                     cooked.push(c);
```

Comment:
Should `cooked` have the same optimization?

Reply:
I think so. I'll patch it tomorrow.